website/project/model.py

```diff
@@ -529,6 +524,8 @@ class Node(GuidStoredObject, AddonModelMixin, IdentifierMixin):
     #: Whether this is a pointer or not
     primary = True
 
+    is_draft_registration = False
```

Comment:
As Sam said, these were mostly a result of Embargo/Retraction/Sanction work. They are similar in name and function but have different roles. I'd not be opposed to adding short docstrings describing their usage as is done in `Node#is_pending_embargo_for_exisiting_registration` but that should be done in another **improvement ticket** in Jira.